os/include/tinyara/fs/ioctl.h

```diff
@@ -94,7 +94,7 @@
 #define _FOTABASE       (0x1900)	/* FOTA ioctl commands */
 #define _GPIOBASE       (0x2000)	/* GPIO ioctl commands */
 #define _TMBASE         (0x2100)	/* Task Management ioctl commands */
-#define _HEAPINFOBASE   (0x2200)	/* Heapinfo ioctl commands */
+#define _MMINFOBASE     (0x2200)	/* Mm ioctl commands */
```

Comment:
I think `mminfo ioctl commands` looks better, because we have `mm` module independently.